### ISSUE TITLE
Detailed steps for using full attic data?

### ISSUE THREAD

**Issue by MaRaSu** [wiktorn/Overpass-API#59]
Has anyone successfully setup an Overpass API server with full (same as Overpass DE i.e. from 2012) attic data (for a certain region / country)?  I noticed an earlier issue #5 touching this topic, with reference to [instructions](https://wiki.openstreetmap.org/wiki/Overpass_API/Installation#Populating_the_DB_with_attic_data) in Overpass wiki. 

However after reading those instructions as well as some further references around the question (e.g. another wiki [entry](https://wiki.openstreetmap.org/wiki/User:Mmd/Overpass_API/install/attic)) I was not sure whether this is feasible and how much effort it would require in terms of learning all the details of the process, obtaining all data files etc.

Has someone done this and if so, could provide simple "for dummies" instructions? Or at least tell what kind of effort to expect to make it work?

(For my use case I'm actually interested mostly on historical "attic" data rather than applying new diffs to stay up-to-date...)

**Comment by wiktorn** [comment]
Probably this is not the best place to ask this question, better would be OSM / Overpass mailing list / Overpass repo.

To do that for specified region, you'll need to:
- download planet file from 2012 and extract area of interest (in docker image there is osmium tool available that can do that for you, you'll only need to specify the geometry of the extract)
- download changes for planet from 2012 up to now, and process them the same way as above
- use both above, to setup Overpass instance

Looks like it will need more CPU time than actual time to set this up.

**Comment by MaRaSu** [comment]
@wiktorn, thanks for a swift reply!

The reason I asked the question here is that I see great value in efforts like these (dockerizing complex apps) and typically the values in the communities around these efforts are focused on how to make app deployments & configuration as convenient as possible rather than expect people to invest lots of time and take a deep dive into a big pile of documentation and test various approaches what might or might not work to figure it out (and with plenty of people repeating the same over a time a great amount of effort goes into waste...).

This repo already has a fantastic solution in providing a simple to deploy overpass api server for current (& future) osm data. It would be even better with a solution supporting full attic data.

I'll take a look at the tips you provided in order to try to figure at least the feasibility of the effort. I guess the search starts by trying to figure out where to find those old planet files & diff files (my understanding on various OSM related data format & file storages is rather shallow, as I have mostly just downloaded latest regional OSM extracts from Geofabrik so far).

**Comment by MaRaSu** [comment]
I should have either done my RTFM or asked a better question ;=) After good 2 hours studying various OSM file formats & other related things I got back to this repo README.md to learn that one can very easily clone full Overpass DE planet DB with all the attic data included. There it was all the time...Now have the container running and under testing. Had to make some room on HDD to fit the whole planet DB (and need to go buy new SSD to improve performance) however my interest in OSM extract to save space is a much lesser concern that having access to full attic data.

@wiktorn, for people like me (with gaps in reading comprehension and patience in reading the docs) it might add value to point out in README.md that with clone & attic options enabled you get full attic data. Somehow when I read it the first time along with some other related discussions in Overpass API wiki I got an impression that with attic option (for this repo's project) you only get "forward" attic data i.e. from DB cloning onwards with new minute diffs providing cumulative attic data.
